process/due-diligence-guidelines.md

```diff
@@ -0,0 +1,142 @@
+('Tech DD') exercise.  Ultimately the voting members of the TOC will,
+on the basis of this and other information, vote for or against the
+inclusion of each project at the relevant time.
+
+## Leading a Technical Due Diligence
+
+### Primary Goals
+
+To enable the voting TOC members to cast an informed vote about a
+project, it is crucial that each member is able to form their own
+opinion as to whether and to what extent the project meets the agreed
+upon [criteria](https://www.cncf.io/projects/graduation-criteria/) for
+inception, incubation or graduation. As the leader of a DD, your job
+is to make sure that they have whatever information they need,
+succinctly and readily available, to form that opinion.
+
+As a secondary goal, it is in the interests of the broader CNCF
+ecosystem that there exists some reasonable degree of consensus across
+the community regarding the inclusion or otherwise of projects at the
+various maturity levels.  Making sure that the relevant information is
+available, and any disagreement or misunderstanding as to it's
+validity are ideally resolved, helps to foster this consensus.
+
+### Where to start
+
+* make sure you're clear on the [TOC Principles](https://github.com/cncf/toc/blob/toc-principles/principles.md),
+  the [project proposal process](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc),
+  the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/)
+  and [desired cloud native properties](https://www.cncf.io/about/charter/) are.  The project sponsor (a member
+  of the TOC) should have assisted in crafting the proposal to explain why it's a good fit for the CNCF. If anything's
+  unclear to you, reach out to the project sponsor or, failing that, the TOC mailing list for advice.   
+* make sure you've read, in detail, the relevant [project proposal](https://github.com/cncf/toc/tree/master/proposals),
+  This will usually be in the form of an [open pull request](https://github.com/cncf/toc/pulls).
+  Consider holding off on commenting on the PR until you've completed the next three steps.
+* take a look at some [previous submissions](https://github.com/cncf/toc/pulls?utf8=%E2%9C%93&q=is%3Apr)
+  (both successful and unsuccessful) to help calibrate your expectations.
+* do as much reading up as you need to (and consult with experts in the specific field) in order to familiarize yourself with the technology
+  landscape in the immediate vicinity of the project (and don't only use the proposal and that project's documentation as a guide in this regard).
+* at this point you should have a very clear technical idea of what exactly the project actually does and does not do, roughly how it compares with and differs from 
+  similar projects in it's technology area, and/or a set of unanswered questions in those regards. 
+* go through the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/) and for each item,
```

Comment:
This makes it sound like the project is expected to meet the graduation criteria already.  

decide for yourself whether or not you believe the project will eventually be able to satisfy the criteria and graduate

Reply:
Agreed, done.